package.json

```diff
@@ -7,7 +7,7 @@
     "@mapbox/node-pre-gyp": "^1.0.9",
     "cheap-ruler": "^3.0.2",
     "mkdirp": "^0.5.6",
-    "nan": "^2.16.0",
+    "nan": "^2.17.0",
```

Comment:
This version seems to contain fixes for that https://github.com/nodejs/nan/commit/7f9ceb80fbc45c9ba1a10e6591ccbef9e8dee6b4#diff-03fb352255b722199246b26ee44cbe9528ab45c581f294c0e8a2fb4ad7ef6ba8L55

Reply:
Not sure what is the root cause, may be Node 18.11.0 which was released recently https://nodejs.org/en/blog/release/v18.11.0/ (and `latest` seems to point to it)